js/main.js

```diff
@@ -0,0 +1,121 @@
+const menu = document.querySelector('.humbergur');
+const nav = document.querySelector('nav ul');
+const navheader = document.querySelector('header');
+
+menu.addEventListener('click', () => {
+  if (menu.firstElementChild.classList.contains('fa-bars')) {
+    nav.classList.toggle('showmenu');
+    menu.firstElementChild.classList.replace('fa-bars', 'fa-xmark');
+    menu.style.color = 'white';
+    navheader.style.background = '#d62828';
+  } else {
+    nav.classList.toggle('showmenu');
+    menu.firstElementChild.classList.replace('fa-xmark', 'fa-bars');
+    navheader.style.background = 'white';
+    menu.style.color = 'black';
+  }
+});
+
+nav.addEventListener('click', () => {
+  nav.classList.toggle('showmenu');
+  nav.style.display = 'hidden';
+  navheader.style.background = 'white';
+  menu.style.color = 'black';
+  menu.firstElementChild.classList.replace('fa-xmark', 'fa-bars');
+});
+
+const staff = [
+  {
+    image: '../img/t7.jpeg',
+    name: 'Jane Bellow',
+    department: 'Senior Teacher',
+    description:
+      'My duties include assigning homework, grading tests, documenting progress and keeping up with parent communication',
+  },
+  {
+    image: '../img/t2.jpeg',
+    name: 'William Great',
+    department: 'School Principle and Senior Administrator',
+    description:
+      'My duties include assigning homework, grading tests, documenting progress and keeping up with parent communication',
+  },
+  {
+    image: '../img/t6.jpeg',
+    name: 'Mercy Greary',
+    department: 'Human Resource Manager',
+    description:
+      'My duties include assigning homework, grading tests, documenting progress and keeping up with parent communication',
+  },
+
+  {
+    image: '../img/s1.jpg',
+    name: 'Hellen Johnson',
+    department: 'Academic Director and Exam Controller',
+    description:
+      'My duties include assigning homework, grading tests, documenting progress and keeping up with parent communication',
+  },
+  {
+    image: '../img/t1.jpeg',
+    name: 'Victor Wise',
+    department: 'Lab Technician and Computer Expert',
+    description:
+      'My duties include assigning homework, grading tests, documenting progress and keeping up with parent communication',
+  },
+  {
+    image: '../img/s2.jpg',
+    name: 'Veronica Mwalia',
+    department: 'Disciplinary Master and Enviromental Manager',
+    description:
+      'My duties include assigning homework, grading tests, documenting progress and keeping up with parent communication',
+  },
+];
+
+const demo = document.getElementById('demo');
+for (let i = 0; i < staff.length; i += 1) {
+  demo.innerHTML += `<div class="toggle"><div class="team">
+  <div class="image-div">
+    <img src="img/rec.jpg" alt="" class="pattern">
```

Comment:
- [ ]  Kindly add alt tags to all images, please do this because if your image is not rendered due to network issues or a broken link, users can still see a text that says what the image about, this make your website more accessible